master-web/master.cfg

```diff
@@ -30,6 +30,7 @@ exec(open("../master-private.cfg").read(), config, { })
 c['title'] = os.getenv('TITLE', default="MariaDB CI")
 c['titleURL'] = os.getenv('TITLE_URL', default="https://github.com/MariaDB/server")
 c['buildbotURL'] = os.getenv('BUILDMASTER_URL', default="https://buildbot.mariadb.org/")
+c['artifactURL'] = os.getenv('ARTIFACT_URL', default="https://ci.mariadb.org/")
```

Comment:
I assume you can't add new entries in c. So, I would suggest saving it to a different variable, but then I am not sure how to get it in the template

Reply:
ok, I'll check the doc...